test/functions/generic/poi/check-gc-reuse-inits.good

```diff
@@ -2,10 +2,10 @@ check-gc-reuse-inits.chpl:38: In function 'u1':
 check-gc-reuse-inits.chpl:39: warning: User.u1
 check-gc-reuse-inits.chpl:19: In initializer:
 check-gc-reuse-inits.chpl:19: warning: User.workIN
-  check-gc-reuse-inits.chpl:41: called as MyRecord.init(param p = 0)
+  check-gc-reuse-inits.chpl:41: called as (MyRecord(0)).init(param p = 0)
 check-gc-reuse-inits.chpl:20: In method 'init=':
 check-gc-reuse-inits.chpl:20: warning: User.workIE
-  check-gc-reuse-inits.chpl:42: called as MyRecord(0).init=(other: MyRecord(0))
+  check-gc-reuse-inits.chpl:42: called as (MyRecord(0)).init=(other: MyRecord(0))
```

Comment:
As I said before, somehow having `0` repeat for an `init=` doesn't bother me that much.

Reply:
Meaning to leave this one alone even though we simplifity `init` for the case above? That is OK with me.